components/common/PanelsCreator.tsx

```diff
@@ -134,6 +135,14 @@ export function PanelsCreator({
                   {Object.keys(filteredPanel).map((panelProp, _index) => (
                     <SummaryList.Row key={_index}>
                       <SummaryList.Key data-cy={`${panelProp}${index}Key`}>
+                        {panelProp === "conditionsOfJoining" ? (
+                          <Label size="s" style={{ float: "right" }}>
+                            <InfoTooltip
+                              tooltipId={`${panelProp}${index}CojInfo`}
+                              content="The Conditions of Joining a Specialty Training Programme is your acknowledgement that you will adhere to the professional responsibilities, including the need to participate actively in the assessment and, where applicable revalidation processes."
+                            ></InfoTooltip>
```

Comment:
Hi @DorisWongNHS InfoTooltip is a React component so self-closing tag i.e. `<InfoTooltip ... />`